diff --git a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
index 929ee5b..8339ec6 100644
--- a/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
+++ b/node_modules/app-builder-lib/out/targets/nsis/NsisTarget.js
@@ -429,7 +429,7 @@ class NsisTarget extends _core().Target {
       defines.UNPACK_DIR_NAME = portableOptions.unpackDirName || (await (0, _builderUtil().executeAppBuilder)(["ksuid"]));
 
       if (portableOptions.splashImage != null) {
-        defines.SPLASH_IMAGE = portableOptions.splashImage;
+        defines.SPLASH_IMAGE = path.resolve(packager.projectDir, portableOptions.splashImage);
       }
     } else {
       await this.configureDefines(oneClick, defines);
diff --git a/node_modules/app-builder-lib/templates/nsis/portable.nsi b/node_modules/app-builder-lib/templates/nsis/portable.nsi
index 2e98dc2..9021447 100644
--- a/node_modules/app-builder-lib/templates/nsis/portable.nsi
+++ b/node_modules/app-builder-lib/templates/nsis/portable.nsi
@@ -8,8 +8,6 @@ AutoCloseWindow True
 RequestExecutionLevel ${REQUEST_EXECUTION_LEVEL}
 
 Function .onInit
-  SetSilent silent
-
   !insertmacro check64BitAndSetRegView
 FunctionEnd
 
@@ -24,10 +22,6 @@ Function .onGUIInit
 FunctionEnd
 
 Section
-  !ifdef SPLASH_IMAGE
-    HideWindow
-  !endif
-
   StrCpy $INSTDIR "$TEMP\${UNPACK_DIR_NAME}"
   RMDir /r $INSTDIR
   SetOutPath $INSTDIR
