diff --git a/node_modules/postcss-load-config/src/index.js b/node_modules/postcss-load-config/src/index.js
index 674bdc6..1fc91fc 100644
--- a/node_modules/postcss-load-config/src/index.js
+++ b/node_modules/postcss-load-config/src/index.js
@@ -56,7 +56,7 @@ const createContext = (ctx) => {
   }, ctx)
 
   if (!ctx.env) {
-    process.env.NODE_ENV = 'development'
+    // process.env.NODE_ENV = 'development'
   }
 
   return ctx
