diff --git a/index.js b/index.js
index 4c3655b324bf4d83cdfeba30eeca68a4fff49721..cfec53510d5d69d19c9e63bfd0e3a86ca4f73442 100644
--- a/index.js
+++ b/index.js
@@ -66,7 +66,17 @@ module.exports = function(content) {
         }
     });

-    sassJs.compile(content, {inputPath: this.resourcePath}, function(result) {
+    // handle empty files
+    if (!content || !content.trim()) {
+        callback(null, content);
+        return;
+    }
+
+    // If we are compiling sass and indentedSyntax isn't set, automatically set it.
+    const ext = path.extname(this.resourcePath);
+    const indentedSyntax = ext && ext.toLowerCase() === '.sass';
+
+    sassJs.compile(content, {inputPath: this.resourcePath, indentedSyntax}, function(result) {
         if (!result.status) {
             callback(null, result.text);
         } else {