diff --git a/node_modules/webpack-dev-server/lib/Server.js b/node_modules/webpack-dev-server/lib/Server.js
index efe8c51..2aef06e 100644
--- a/node_modules/webpack-dev-server/lib/Server.js
+++ b/node_modules/webpack-dev-server/lib/Server.js
@@ -54,7 +54,7 @@ class Server {
       throw new Error("'filename' option must be set in lazy mode.");
     }
 
-    validateOptions(schema, options, 'webpack Dev Server');
+    // validateOptions(schema, options, 'webpack Dev Server');
 
     this.compiler = compiler;
     this.options = options;
