diff --git a/node_modules/react-refresh/cjs/react-refresh-runtime.development.js b/node_modules/react-refresh/cjs/react-refresh-runtime.development.js
index e17c488..afa980a 100644
--- a/node_modules/react-refresh/cjs/react-refresh-runtime.development.js
+++ b/node_modules/react-refresh/cjs/react-refresh-runtime.development.js
@@ -621,10 +621,10 @@ function isLikelyComponentType(type) {
             } // eslint-disable-next-line no-proto
 
 
-            if (type.prototype.__proto__ !== Object.prototype) {
-              // It has a superclass.
-              return false;
-            } // Pass through.
+            // if (type.prototype.__proto__ !== Object.prototype) {
+            //   // It has a superclass.
+            //   return false;
+            // } // Pass through.
             // This looks like a regular function with empty prototype.
 
           } // For plain functions and arrows, use name as a heuristic.
